vm/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Change `execution` module to use `execution::Config`
```

Comment:
Missing issue

Reply:
It's a refactor PR required by a third issue. 
How do you suggest to handle? Should I create an issue related to the non maintainability of that?